components/frontend/src/locale.js

```diff
@@ -0,0 +1,30 @@
+import "dayjs/locale/en-gb"
+import "dayjs/locale/nl"
+
+import dayjs from "dayjs"
+import localeData from "dayjs/plugin/localeData"
+import updateLocale from "dayjs/plugin/updateLocale"
+import { string } from "prop-types"
+
+export function adapterLocale(language) {
+    // Quick and dirty solution while we wait for a more generic solution to finding the user's locale
+    // https://github.com/iamkun/dayjs/issues/732
+    let locale = "en-GB"
+    if (language === "nl-NL") {
+        // Use the Dutch locale, but keep the language English
```

Comment:
Shouldn't we keep the language English for any locale? We're writing everything in English so it would be very weird if random words might be translated due to some locale

Reply:
Yes, what I was thinking exactly. And that is what the code is doing: we always use en-GB, except when the user's locale is nl-NL. Then we use the Dutch locale with the language parts (weekday names and months) reset to English. We can't generically do what we do for Dutch because of the issue mentioned in the code.